src/stores/views.ts

```diff
@@ -691,11 +709,14 @@ export function viewAddFilterItemsToRow(
     newRows.splice(insertIndex + 1, 0, returnRow);
   } else {
     // Find the layer in the row or create one if needed
+    const appropriateLayerInRow: Layer | undefined = row.layers.find(iterLayer => iterLayer.chartType === typeName);
```

Comment:
This logic interferes with the behavior below where a user specifies new layer (which would intentionally set layer to null)

Reply:
Removing this logic - was another remnant of trying to include external event types 'automatically' to the filter when no row/layer is explicitly defined for use. No longer needed with the rework to instead use the timeline items tab for filtering!